modules/opengl/src/geometry/meshgl.cpp

```diff
@@ -95,13 +95,13 @@ void MeshGL::update(bool editable) {
 
     bufferArray_.bind();
     Mesh* owner = this->getOwner();
-    // update all buffers except index buffers, i.e. position, color, normals, etc.)
+    // update regular buffers except index buffers
```

Comment:
What is a regular buffer?

Reply:
everything beside an index buffer, might change it back to "all".